decidim-design/config/locales/en.yml

```diff
@@ -0,0 +1,5 @@
+---
+en:
+  decidim:
+    home:
+      home_header: Decidim Design Guide
```

Comment:
```suggestion
en:
  decidim:
    design:
      home:
        index: 
          home_header: Decidim Design Guide
```

Reply:
@greenwoodt and here its how it infers it, by following the filepath/controllers definition:. a key `.home_header` in `decidim-design/app/views/**decidim/design/home/index**.html.erb` becomes `decidim.design.home.index.home_header`.  
And as we're following this convention, instead of the full key name (`decidim.design.home.index.home_header`) we can get away with just adding the last part (`.home_header`)

Reply:
> @greenwoodt and here its how it infers it, by following the filepath/controllers definition:. a key `.home_header` in `decidim-design/app/views/**decidim/design/home/index**.html.erb` becomes `decidim.design.home.index.home_header`. And as we're following this convention, instead of the full key name (`decidim.design.home.index.home_header`) we can get away with just adding the last part (`.home_header`)

Understood!!